src/Network/Network/PrivateLinkService/PrivateLinkServiceProvider/ProviderConfiguration.cs

```diff
@@ -30,6 +30,8 @@ static ProviderConfiguration()
             RegisterConfiguration("Microsoft.DBforMariaDB/servers", "2018-06-01", true, true);
             RegisterConfiguration("Microsoft.DBforMySQL/servers", "2018-06-01", true, true);
             RegisterConfiguration("Microsoft.DBforPostgreSQL/servers", "2018-06-01", true, true);
+            RegisterConfiguration("Microsoft.DesktopVirtualization/hostpools", "2021-09-03-preview", true, true);
```

Comment:
```suggestion
            RegisterConfiguration("Microsoft.DesktopVirtualization/hostpools", "2021-09-03-preview", true, false);''
```
As there is not API for "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DesktopVirtualization/hostPools/{hostPoolName}/privateLinkResources/{"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DesktopVirtualization/hostPools/{hostPoolName}/privateLinkResourceName}"